orchestra/model/configuration.py

```diff
@@ -66,8 +67,7 @@ def __init__(self, args):
 
         self._create_default_user_options()
 
-        self.generated_yaml = run_ytt(self.orchestra_dotdir, use_cache=not args.no_config_cache)
-        self.parsed_yaml = yaml.safe_load(self.generated_yaml)
+        self.parsed_yaml, self.generated_yaml = self.load_configuration(use_cache=not args.no_config_cache)
```

Comment:
What's the purpose of `self.generated_yaml`?

Reply:
It's used for `orchestra dumpconfig`

Reply:
It's a bit of a waste to load every time the yaml just for `dumpconfig`.
Can't we `cat cache.yml` only in case `dumpconfig` command is invoked?